configs/example_configs_validation.py

```diff
@@ -0,0 +1,32 @@
+import os
+import pathlib
+import sys
+
+import yaml
+
+from google.protobuf.json_format import ParseError
+
+sys.path = [p for p in sys.path if not p.endswith('bazel_tools')]
+
+from tools.config_validation.validate_fragment import ValidateFragment
+
+
+def main():
+  errors = []
+  for arg in sys.argv[1:]:
+    try:
+      ValidateFragment("envoy.config.bootstrap.v3.Bootstrap",
+                       yaml.safe_load(pathlib.Path(arg).read_text()))
+    except (ParseError, KeyError) as e:
+      errors.append(arg)
+      print()
+      print(f"ERROR (validation failed): {arg}")
```

Comment:
Nit: prefer `\n` or a template than using `print()` for ASCII format effects

Reply:
ill update...